### PR TITLE
Bernard/use UI library

### PR DESCRIPTION
**Pull request by bernarduskrishna**
### Summary 
Use pinus-ui-library.

### Detail
UI should still look as close as possible to the current pinusonline.

### Pictures / Gif 
todo: mobile UI is broken now due to the header.
image
In the previous navbar, when it's on mobile, the navbar would change completely (use dropdown). Probably add this support for pinus-ui-library?

### Checks 
- [x] Simon

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (simonjulianl) [inline .vscode/settings.json:1]: oh btw dont push this to the repo, this is internal vscode setting

File: src/components/Page/links.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  alt?: boolean;
+}
+
+export const navLinks: NavLink[] = [
+  { title: "About", slug: "about" },
+  { title: "Admissions", slug: "admissions" },
+  { title: "Events", slug: "events" },
+  { title: "Visit Us", slug: "visit" },
+  { title: "Contact Us", slug: "contact" },
+  // { title: "Aksara", slug: "aksara", alt: true },
```
Comment (simonjulianl) [inline src/components/Page/links.tsx:13]: Should we remove this?

**Review by simonjulianl — CHANGES_REQUESTED** [review]
Just some minor changes

File: src/pageContent/ContactUs/index.tsx [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import React, { useState } from "react";
 
-import Button from "src/components/Button";
+import { Button } from "pinus-ui-library";
 import Input from "src/components/Input";
```
Comment (simonjulianl) [inline src/pageContent/ContactUs/index.tsx:4]: Is this used or required ?

File: src/pageContent/Home/index.tsx [hunk 4]
```diff
@@ -1,24 +1,32 @@
...
-import Carousel from "src/components/Carousel";
-import Text from "src/components/Text";
+import { Text, Button } from "pinus-ui-library";
 import * as S from "./styles";
 
 const HomeContent: React.FC = () => {
   const router = useRouter();
 
+  {
+    /* TODO: Why TF are all other non-Text texts redddd */
```
Comment (simonjulianl) [inline src/pageContent/Home/index.tsx:10]: KEKEKEK, btw dont need to add braces

File: src/pageContent/ContactUs/index.tsx (hunk 3 above)
Comment (bernarduskrishna) [reply src/pageContent/ContactUs/index.tsx:4]: ohhh yea i think this one required lol. haven't changed to the UI Library one

Comment (bernarduskrishna) [reply src/pageContent/ContactUs/index.tsx]: done

**Review by simonjulianl — APPROVED** [review]
LGTM

**Pull request merged**